src/main/scala/org/globalforestwatch/summarystats/carbon_sensitivity/CarbonSensitivitySummary.scala

```diff
@@ -78,7 +78,7 @@ object CarbonSensitivitySummary {
         val lossYearLegalAmazon: Integer = raster.tile.lossLegalAmazon.getData(col, row)
         val prodesLegalAmazonExtent2000: Boolean = raster.tile.prodesLegalAmazonExtent2000.getData(col, row)
         val tropicLatitudeExtent: Boolean = raster.tile.tropicLatitudeExtent.getData(col, row)
-        val burnYearHansenLoss: Integer = raster.tile.burnYearHansenLoss.getData(col, row)
+        val burnYearHansenLoss: Boolean = raster.tile.treeCoverLossFromFires.getData(col, row)
```

Comment:
Rename this val (maybe to isBurnLoss?), now that it is a boolean, and not a year?

Reply:
In other packages it's named treeCoverLossFromFires, so I'm going to use that. This burnYearHansenLoss was a legacy from a previous layer I used.